collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, configuration=None, name=None):
         self.url = self.configuration.get('url', DEFAULT_URL)
         self.collect_alarm_values = bool(self.configuration.get('collect_alarm_values', DEFAULT_COLLECT_ALARM_VALUES))
         self.collected_dims = {'alarms': set(), 'values': set()}
+        self.alarm_contains_words = self.configuration.get('alarm_contains_words', DEFAULT_ALARM_CONTAINS_WORDS)
```

Comment:
I suggest splitting and, optionally, trimming spaces (e.g. hadnle `cpu, load`) here

Reply:
Will do

Reply:
done